yt_dlp/extractor/bibeltv.py

```diff
@@ -1,27 +1,204 @@
+            f'{self.API_URL}/video/{video_id}', crn_id, headers={'Authorization': self.AUTH_TOKEN}
+        )
+        if json_data.get('status') != 'success':
+            raise ExtractorError('Failed to load JSON metadata')
+
+        formats, subtitles = self._extract_formats_and_subtitles(
+            try_get(json_data, lambda x: x['video']['videoUrls'], list) or [], crn_id)
+
+        return {
+            'id': crn_id,
+            'title': data.get('title'),
+            'description': data.get('description'),
+            'duration': try_get(data, lambda x: x['duration'] / 1000),
+            'timestamp': parse_iso8601(data.get('schedulingStart')),
+            'thumbnails': [{'url': url} for url in {image['url'] for image in data.get('images', ())}],
+            'formats': formats,
+            'subtitles': subtitles,
+            'season_number': data.get('seasonNumber'),
+            'episode_number': data.get('episodeNumber'),
+            'view_count': data.get('viewCount'),
+            'like_count': data.get('likeCount'),
+        }
+
+    def _real_extract(self, url):
+        crn_id = self._match_id(url)
+        webpage = self._download_webpage(url, crn_id)
+        nextjs_data = self._search_nextjs_data(webpage, crn_id)
+        video_data = try_get(nextjs_data,
+                             lambda x: x['props']['pageProps']['videoPageData']['videos'][0], dict)
+        if not video_data:
+            raise ExtractorError('Missing video data.')
+
+        return self._extract_video(video_data)
+
+
+class BibelTvSerienIE(BibelTVIE):
+    IE_DESC = 'BibelTV'
+    _VALID_URL = r'https?://(?:www\.)?bibeltv\.de/mediathek/serien/(?:crn/)?(?P<id>\d+)'
+    _RETURN_TYPE = 'playlist'
+
+    IE_NAME = 'bibeltv:serien'
+
+    _TESTS = [{
+        'url': 'https://www.bibeltv.de/mediathek/serien/333485-ein-wunder-fuer-jeden-tag',
+        'playlist_mincount': 400,
+        'info_dict': {
+            'id': '333485',
+            'title': 'Ein Wunder für jeden Tag',
+            'description': 'Tägliche Kurzandacht mit Déborah Rosenkranz.',
+        },
+    }]
 
     def _real_extract(self, url):
         crn_id = self._match_id(url)
-        return self.url_result(
-            self.BRIGHTCOVE_URL_TEMPLATE % crn_id, 'BrightcoveNew')
+        webpage = self._download_webpage(url, crn_id)
+        nextjs_data = self._search_nextjs_data(webpage, crn_id)
+        serie_data = try_get(nextjs_data,
+                             lambda x: x['props']['pageProps']['seriePageData'], dict)
+        if not serie_data:
+            raise ExtractorError('Missing series data.')
+
+        video_data = serie_data.get('videos', ())
+
+        return self.playlist_result(
+            LazyList(map(self._extract_url_info, video_data)),
+            playlist_id=crn_id,
+            playlist_title=serie_data.get('title'),
+            playlist_description=clean_html(serie_data.get('description')),
+            playlist_count=len(video_data),
+        )
```

Comment:
```suggestion
        return self.playlist_result(
            list(map(self._extract_url_info, video_data)), crn_id,
            serie_data.get('title'), clean_html(serie_data.get('description')))
```
Since there is no API calls, there's no need for it to be lazylist